arangod/RocksDBEngine/RocksDBIncrementalSync.cpp

```diff
@@ -889,9 +887,10 @@ Result handleSyncKeysRocksDB(DatabaseInitialSyncer& syncer,
               auto [documentId, revisionId] = lookupResult;
 
               tempBuilder.clear();
-              r = physical->lookupDocument(
-                  *trx, documentId, tempBuilder, /*readCache*/ true,
-                  /*fillCache*/ false, ReadOwnWrites::yes);
+              r = physical->lookup(trx.get(), documentId, callback,
+                                   {.readCache = true,
```

Comment:
nit: `.readCache = true` seems to be the default and doesn't need to be mentioned explicitly here.
All previous callers of `lookup(...)` at least did not specify it explicity, so this place here is a little inconsistent with the previous occurrences.